kpops/utils/docstring.py

```diff
@@ -0,0 +1,83 @@
+import logging
+
+from pydantic import BaseModel
+
+log = logging.getLogger("docstring_utils")
+
+
+def describe_attr(name: str, docstr: str) -> str:
+    """Read attribute description from class docstring
+
+    Works only with reStructuredText docstrings.
+
+    :param name: Attribute name
+    :type name: str
+    :param docstr: Docstring from which to read. Note that the class' docstring is stored in attr `__doc__`
+    :type docstr: str
+    :returns: Description of the class attribute read from the class docstring
+    :rtype: str
+    """
+    docstr = docstr.partition(f":param {name}:")[2]
+
+    docstr = _trim_description_end(docstr)
+
+    return docstr
+
+
+def describe_class(class_: type[BaseModel] | str) -> str:
```

Comment:
I can only find one usage where the `class_` argument is actually a class (`type[BaseModel]`). I would suggest passing there `model.__doc__` as well so that this function has less responsibilities. Then you could change the function signature also to `docstr: str | None`

Reply:
I did, you are right